wca-guidelines.md

```diff
@@ -201,6 +201,7 @@ To be more informative, each Guideline is classified using one of the following
 
 - E2a1+) [RECOMMENDATION] If a judge finds a submitted solution that does not appear to have information to identify the attempt, they should check both sides of the paper.
 - E2b+) [CLARIFICATION] A competitor may choose to stop an attempt early by handing in a solution before the time limit.
+- E2b++) [ADDITION] A competitor may join a Fewest Moves attempt that is aleready in progress, at the discretion of the Delegate. In this case, their time limit is the time remaining (i.e. they must finish when other competitors reach their 60 minute time limit). Exception: a competitor must not be permitted to join a Fewest Moves attempt if any competitor has finished their attempt already (see [Guideline E2b+](guidelines:guideline:E2b+)), or if there is any other risk that information about the scramble may have become known to the competitor who wishes to join.
```

Comment:
```suggestion
- E2b++) [ADDITION] A competitor may join a 3x3x3 Fewest Moves attempt that is aleready in progress, at the discretion of the WCA Delegate. In this case, their time limit is the time remaining (i.e. they must finish when other competitors reach their 60 minute time limit). Exception: a competitor must not be permitted to join a 3x3x3 Fewest Moves attempt if any competitor has finished their attempt already (see [Guideline E2b+](guidelines:guideline:E2b+)), or if there is any other risk that information about the scramble may have become known to the competitor who wishes to join.
```

Reply:
Ah, good catch. I'm still thrown off by that, but your correction is the right choice for consistency!